src/components/lyrics/offset-control.tsx

```
@@ -27,6 +33,21 @@ const OffsetControl = ({ value, min = -5000, max = 5000, onChange, onOpenChange
     [onOpenChange],
   );
 
+  const handleCustomChange = (e: React.ChangeEvent<HTMLInputElement> | string) => {
+    const val = typeof e === 'string' ? e : e.target.value;
```

ESLint

[error] Replace `'string'` with `"string"`

(prettier/prettier)